tools/src/test/python/dlpx/virtualization/_internal/commands/test_codegen.py

```diff
@@ -363,12 +363,15 @@ def test_copy_generated_to_dir_success(tmpdir, popen_helper):
 
     @staticmethod
     def test_copy_generated_to_dir_fail(tmpdir):
-        src_dir = '/not/a/real/dir'
+        src_dir = os.path.join('fake', 'dir')
         # dst_dir needs to be real so that making the dir inside it works.
         dst_dir = tmpdir.strpath
 
         with pytest.raises(OSError) as err_info:
             codegen._copy_generated_to_dir(src_dir, dst_dir)
 
-        assert err_info.value.strerror == 'No such file or directory'
-        assert err_info.value.filename.startswith('/not/a/real/dir')
+        assert (
+            err_info.value.strerror == 'No such file or directory' or
+            err_info.value.strerror == 'The system cannot find the path specified'
```

Comment:
is one of these the windows error and the other the unix? can we just leave a comment saying which os throws what so that in the future we know why we'd expect different error messages?

Or better yet maybe we can assert if os.name == 'nt' (? im just using the if statement you had above) and the expected error message that would make it explicit so that if something fails we'll know pretty quickly why?